integration-test/features/organizations.feature

```diff
@@ -42,3 +42,8 @@ Feature: All about Organizations
     When the organization set the status to "DISABLED"
     Then the organization gets the status code 200
     And the status is "DISABLED" in the organization's details
+    
+  Scenario: An organization try to create an enrollment to a service without some mandatory parameters
+    Given the organization "777777" with the service "service-1" without mandatory parameters
+    When the organization "777777" creates a creditor institution with enrollment to service
```

Comment:
the same parameter should not be repeated between steps.
```
    Given the organization "777777" with the service "service-1" without mandatory parameters
    When the organization "777777" creates a creditor institution with enrollment to service
```
in to something like
```
    Given the organization "777777"
    When the organization enrolls to service "service-1" without mandatory parameters
```

or better using in this case `Scenario Outline` with `Example`